package.json

```diff
@@ -155,6 +155,7 @@
     "globalthis": "^1.0.2",
     "hammerjs": "^2.0.8",
     "hash-wasm": "^4.4.1",
+    "headroom.js": "^0.12.0",
```

Comment:
Not a new addition, we already load this using `third_party.js`. We will remove this library from `third_party` once ajs is removed.